tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -3127,84 +3451,150 @@ def create_zeros_like_shape_F(n,m):
+
     f_int_float = epyccel(create_zeros_like_val_float, language = language)
-    assert(isclose(     f_int_float()       ,      create_zeros_like_val_float(), rtol=RTOL, atol=ATOL))
     assert matching_types(f_int_float(), create_zeros_like_val_float())
 
+    f_complex_auto = epyccel(create_zeros_like_val_complex_auto, language = language)
+    assert matching_types(f_complex_auto(), create_zeros_like_val_complex_auto())
+
     f_int_complex = epyccel(create_zeros_like_val_complex, language = language)
-    assert(isclose(     f_int_complex()     ,      create_zeros_like_val_complex(), rtol=RTOL, atol=ATOL))
     assert matching_types(f_int_complex(), create_zeros_like_val_complex())
+    
+    f_int32_auto   = epyccel(create_zeros_like_val_int32_auto, language = language)
+    assert matching_types(f_int32_auto(), create_zeros_like_val_int32_auto())
 
     f_real_int32   = epyccel(create_zeros_like_val_int32, language = language)
-    assert(     f_real_int32()       ==      create_zeros_like_val_int32())
     assert matching_types(f_real_int32(), create_zeros_like_val_int32())
 
+    f_float32_auto   = epyccel(create_zeros_like_val_float32_auto, language = language)
+    assert matching_types(f_float32_auto(), create_zeros_like_val_float32_auto())
+
     f_real_float32   = epyccel(create_zeros_like_val_float32, language = language)
-    assert(isclose(     f_real_float32()    ,      create_zeros_like_val_float32(), rtol=RTOL, atol=ATOL))
     assert matching_types(f_real_float32(), create_zeros_like_val_float32())
 
+    f_float64_auto   = epyccel(create_zeros_like_val_float64_auto, language = language)
+    assert matching_types(f_float64_auto(), create_zeros_like_val_float64_auto())
+
     f_real_float64   = epyccel(create_zeros_like_val_float64, language = language)
-    assert(isclose(     f_real_float64()    ,      create_zeros_like_val_float64(), rtol=RTOL, atol=ATOL))
     assert matching_types(f_real_float64(), create_zeros_like_val_float64())
 
+    f_complex64_auto   = epyccel(create_zeros_like_val_complex64_auto, language = language)
+    print(type(f_complex64_auto()), type(create_zeros_like_val_complex64_auto()))
+    assert matching_types(f_complex64_auto(), create_zeros_like_val_complex64_auto())
+
     f_real_complex64   = epyccel(create_zeros_like_val_complex64, language = language)
-    assert(isclose(     f_real_complex64()  ,      create_zeros_like_val_complex64(), rtol=RTOL, atol=ATOL))
     assert matching_types(f_real_complex64(), create_zeros_like_val_complex64())
 
+    f_complex128_auto   = epyccel(create_zeros_like_val_complex128_auto, language = language)
+    assert matching_types(f_complex128_auto(), create_zeros_like_val_complex128_auto())
+
     f_real_complex128   = epyccel(create_zeros_like_val_complex128, language = language)
-    assert(isclose(     f_real_complex128() ,      create_zeros_like_val_complex128(), rtol=RTOL, atol=ATOL))
```

Comment:
Why did you decide to remove the value tests?

Reply:
that was a mistake, ill add it back.

Reply:
It primarily checks the type, but extra tests are never a problem ;)